packages/components/ng-ui-router-breadcrumb/src/ng-ui-router-breadcrumb.html

```diff
@@ -7,7 +7,7 @@
         ></a>
         <span
             data-ng-if="!(element.url && !element.active && !$last)"
-            data-ng-bind="element.name"
+            data-ng-bind="$ctrl.convertDomainToPunycode(element.name)"
```

Comment:
Does this work as it is a static class method ?
Maybe you should use constructor to access this method
```suggestion
            data-ng-bind="$ctrl.constructor.convertDomainToPunycode(element.name)"
```

Reply:
Yes it worked and was tested (cf Jira). Let me check another time :)

Reply:
Seems ok also in the deployed version:
![image](https://github.com/ovh/manager/assets/8767496/a275c6cd-0bb9-4457-8b6e-130da63917c3)

Reply:
You're right, thks, let me correct this!